internal/plugin/plugin.go

```diff
@@ -15,28 +15,35 @@
 package plugin
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"os/exec"
+	"path"
 
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/config"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/log"
 	"github.com/spf13/cobra"
 )
 
+const (
+	ExtraPluginDirectoryEnvKey = "ATLAS_CLI_EXTRA_PLUGIN_DIRECTORY"
+)
+
 func GetAllValidPlugins(existingCommands []*cobra.Command) []*Plugin {
 	var manifests []*Manifest
 
 	// Load manifests from plugin directories
-	if loadedManifests, err := getManifestsFromPluginDirectory("./plugins"); err != nil {
-		logPluginWarning(`could not load manifests from directory "./plugins" because of error: %s`, err.Error())
-	} else {
-		manifests = append(manifests, loadedManifests...)
+	if defaultPluginDirectory, err := GetDefaultPluginDirectory(); err == nil {
+		if loadedManifests, err := getManifestsFromPluginsDirectory(defaultPluginDirectory); err != nil {
+			logPluginWarning(`could not load manifests from directory "%s" because of error: %s`, defaultPluginDirectory, err.Error())
+		} else {
+			manifests = append(manifests, loadedManifests...)
```

Comment:
What's the point of this else branch? Can there still be loadedManifests when there's an error?

Reply:
changed that now

Reply:
I'm sorry, I must have read the original code wrong.
I thought you were adding `manifests = append(manifests, loadedManifests...)` in both branches, which you were not. Can you revert this change?

Reply:
no worries, just reverted it :)